.gitignore

```diff
@@ -7,3 +7,4 @@ src/main/resources/spells.defaults.yml
 src/main/resources/wands.defaults.yml
 **/original_skin.png
 **/mc_creds.php
+*.project
```

Comment:
Shouldn't this be just `.project`, as you want to get rid of all files that are named exactly that.